test cases/rust/12 bindgen/cpp/meson.build

```diff
@@ -0,0 +1,19 @@
+# SPDX-license-identifer: Apache-2.0
+# Copyright © 2021-2023 Intel Corporation
```

Comment:
2024

Reply:
The code in the commit was written and copyrighted in 2024 so this is fine. :)